crates/server/src/admin/storage/jwt_secret.rs

```diff
@@ -0,0 +1,76 @@
+use calimero_store::entry::{Entry, Json};
+use calimero_store::key::Generic;
+use calimero_store::Store;
+use rand::Rng;
+use serde::{Deserialize, Serialize};
+struct JwtTokenSecretEntry {
+    key: Generic,
+}
+
+impl Entry for JwtTokenSecretEntry {
+    type Key = Generic;
+    type Codec = Json;
+    type DataType<'a> = JwtTokenSecret;
+
+    fn key(&self) -> &Self::Key {
+        &self.key
+    }
+}
+
+impl JwtTokenSecretEntry {
+    fn new() -> Self {
+        Self {
+            key: Generic::new(*b"jwt_salt::server", [0; 32]),
```

Comment:
keep in mind you can store multiple entries under the fragment here, up to you to determine what's a good discriminant.

since the refresh token includes information about the executor and the expiry date, you can use that as the fragment specifier